conda/base/context.py

```diff
@@ -338,10 +338,21 @@ class Context(Configuration):
     update_modifier = ParameterLoader(PrimitiveParameter(UpdateModifier.UPDATE_SPECS))
     sat_solver = ParameterLoader(PrimitiveParameter(SatSolverChoice.PYCOSAT))
     solver_ignore_timestamps = ParameterLoader(PrimitiveParameter(False))
-    experimental_solver = ParameterLoader(
-        PrimitiveParameter(ExperimentalSolverChoice.CLASSIC, element_type=ExperimentalSolverChoice)
+    solver = ParameterLoader(
+        PrimitiveParameter(SolverChoice.CLASSIC, element_type=SolverChoice),
+        aliases=('experimental_solver',),
     )
 
+    @property
+    def experimental_solver(self):
+        # TODO: Remove in a later release
+        warnings.warn(
+            "'context.experimental_solver' is deprecated and will be removed. "
+            "Please use 'context.solver' instead.",
+            DeprecationWarning
```

Comment:
```suggestion
            PendingDeprecationWarning
```

Reply:
Python says https://docs.python.org/3/library/exceptions.html#PendingDeprecationWarning `This class is rarely used as emitting a warning about a possible upcoming deprecation is unusual, and [DeprecationWarning](https://docs.python.org/3/library/exceptions.html#DeprecationWarning) is preferred for already active deprecations.`